src/components/cc-logs-instances/cc-logs-instances.js

```diff
@@ -0,0 +1,755 @@
+    `;
+  }
+
+  /**
+   * @param {Array<Instance>} instances
+   */
+  _renderRunningInstances (instances) {
+    if (instances.length === 0) {
+      return html`
+        <fieldset class="section section--running">
+          ${this._renderHeader({
+            title: i18n('cc-logs-instances.running.header'),
+            icon: iconHeaderRunning,
+          })}
+          <div class="empty">
+            ${i18n('cc-logs-instances.running.empty')}
+          </div>
+        </fieldset>
+      `;
+    }
+
+    return html`
+      <fieldset class="section section--running">
+        ${this._renderHeader({
+          title: i18n('cc-logs-instances.running.header'),
+          icon: iconHeaderRunning,
+          commit: instances[0].deployment.commitId,
+        })}
+        <div class="section-content instances">
+          ${this._renderInstances(instances, false)}
+        </div>
+      </fieldset>
+    `;
+  }
+
+  /**
+   * @param {Array<Instance>} instances
+   */
+  _renderStoppingInstances (instances) {
+    if (instances.length === 0) {
+      return '';
+    }
+
+    return html`
+      <fieldset class="section section--stopping">
+        ${this._renderHeader({
+          title: i18n('cc-logs-instances.stopping.header'),
+          icon: iconHeaderStopping,
+          commit: instances[0].deployment.commitId,
+        })}
+        <div class="section-content instances">
+          ${this._renderInstances(instances, false)}
+        </div>
+      </fieldset>
+    `;
+  }
+
+  /**
+   * @param {Array<Instance>} instances
+   */
+  _renderDeletedInstances (instances) {
+    if (instances.length === 0) {
+      return '';
+    }
+
+    return html`
+      <fieldset class="section section--deleted">
+        ${this._renderHeader({
+          title: i18n('cc-logs-instances.deleted.header'),
+          icon: iconHeaderDeleted,
+        })}
+        ${this._renderInstancesGroupedByDeployment(instances, false)}
+      </fieldset>
+    `;
+  }
+
+  /**
+   * @param {Array<GhostInstance>|null} instances
+   */
+  _renderGhostInstances (instances) {
+    if (instances == null || instances.length === 0) {
+      return '';
+    }
+
+    return html`
+      <fieldset class="section section--ghost">
+        ${this._renderHeader({
+          title: i18n('cc-logs-instances.ghost.header'),
+          icon: iconHeaderGhost,
+        })}
+
+        <div class="section-content">
+          <cc-notice intent="info" message="${i18n('cc-logs-instances.ghost.notice')}"></cc-notice>
+          <div class="instances instances--ghost">
+            ${instances.sort(GHOST_INSTANCE_SORT_ORDER).map((instance) => html`
+            <label class="instance" for="${instance.id}">
+              <input type="checkbox"
+                     id="${instance.id}"
+                     .checked=${this._isSelected(instance.id)}
+                     @change=${this._onInstanceClick}>
+              <span class="instance-id">${instance.id}</span>
+            </label>
```

Comment:
question: to we really need the `for` and `id` if we wrap the input inside the label?

question: should we prefix the raw instance ID with `instance-`?